gateway/tests/test_scim.py

```diff
@@ -519,7 +527,7 @@ def test_delete_fdo_device(client: FlaskClient, api_key):
             "deviceDisplayName": "Generic FDO Device",
             "adminState": True,
             "urn:ietf:params:scim:schemas:extension:fido-device-onboard:2.0:Device": {
-                "fdoVoucher": "there-should-be-a-voucher-here"
+                "fdoOwnerVoucher": voucher
```

Comment:
Can you extend the test to mock the FDO_OWNER_URI and intercept the request to verify that the request is what we expect? (We do something similar in the python sdk tests so you can take a look at that). 

I ask this because we are now sending a PEM and I would like to verify that the request is sent correctly with a test.

Reply:
I think that's possible, but I'd suggest opening an issue for it.

Reply:
Opened #70

Reply:
I actually did a live test with the FDO OSS base.  I agree that building such a test is a good idea, but that's not for this ticket.